### PR TITLE
Auto-suggest experiment

### PR DESCRIPTION
**Pull request by yattias**
- Adding auto-suggest experiment available behind a query string

https://www.loom.com/share/69e1431d5e934b3f8e63977fe123d3af

### CODE REVIEW

File: components/Author/AuthorList.tsx [hunk 1]
```diff
@@ -30,6 +30,42 @@ const Author = ({ author }: { author: AuthorProfile }) => {
...
+
+  if (authors.length > 1) {
+    // Last author is the second most important author
+    primaryAuthors.push(authors[authors.length - 1]);
+  }
+
+  if (authors.length > 2) {
+    showEtAllText = true;
+  }
+  console.log("authors", authors);
```
Comment (koutst) [inline components/Author/AuthorList.tsx:53]: Are these logs meant to be here?

File: components/SearchSuggestion/lib/api.ts [hunk 2]
```diff
@@ -4,7 +4,41 @@ import {
...
+  Suggestion,
 } from "./types";
+import { NullableString } from "~/config/types/root_types";
+
+export const fetchAllSuggestions = (
+  query: NullableString
+): Promise<Suggestion[]> => {
+  const url = `${API.BASE_URL}search/combined-suggest/?query=${query}`;
+
+  const sugestions: Suggestion[] = [];
```
Comment (koutst) [inline components/SearchSuggestion/lib/api.ts:17]: nit: sugestions -> suggestions

**Review by koutst — APPROVED** [review]
LGTM

File: components/SearchSuggestion/lib/api.ts (hunk 2 above)
Comment (yattias) [reply components/SearchSuggestion/lib/api.ts:17]: Good Call!

**Pull request merged**